### PR TITLE
[DOCS]: Port over "How to host and share DataDocs on Azure" from RTD to Docusaurus 

### PR DESCRIPTION
**Pull request by cdkini**
Please annotate your PR title to describe what the PR does, then give a brief bulleted description of your PR below. PR titles should begin with [BUGFIX], [FEATURE],  [DOCS], or [MAINTENANCE]. If a new feature introduces breaking changes for the Great Expectations API or configuration files, please also add [BREAKING]. You can read about the tags in our [contributor checklist](https://docs.greatexpectations.io/en/latest/contributing/contribution_checklist.html).

Changes proposed in this pull request:
- Port over `.rst` to `.md`
-
-

After submitting your PR, CI checks will run and @tiny-tim-bot will check for your CLA signature.

For a PR with nontrivial changes, we review with both design-centric and code-centric lenses.

In a design review, we aim to ensure that the PR is consistent with our relationship to the open source community, with our software architecture and abstractions, and with our users' needs and expectations. That review often starts well before a PR, for example in github issues or slack, so please link to relevant conversations in notes below to help reviewers understand and approve your PR more quickly (e.g. `closes #123`).

Previous Design Review notes:
-
-
-

### Definition of Done
Please delete options that are not relevant.

- [ ] My code follows the Great Expectations [style guide](https://docs.greatexpectations.io/en/latest/contributing/style_guide.html?highlight=style%20guide)
- [ ] I have performed a [self-review](https://docs.greatexpectations.io/en/latest/contributing/contribution_checklist.html?highlight=checklist) of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added [unit tests](https://docs.greatexpectations.io/en/latest/contributing/testing.html#contributing-testing-writing-unit-tests) where applicable and made sure that new and existing tests are passing.
- [ ] I have run any local integration tests and made sure that nothing is broken.

Thank you for submitting!

### CODE REVIEW

**Comment by cdkini** [comment]
Screenshot 2021-07-09 at 14-35-16 How to host and share Data Docs on Azure Blob Storage Great Expectations
Screenshot 2021-07-09 at 14-36-07 How to host and share Data Docs on Azure Blob Storage — great_expectations documentation

File: docs/guides/setup/configuring-data-docs/how-to-host-and-share-data-docs-on-azure-blob-storage.md [hunk 1]
```diff
@@ -1,5 +1,103 @@
...
+  ```yaml
+ AZURE_STORAGE_CONNECTION_STRING: "DefaultEndpointsProtocol=https;EndpointSuffix=core.windows.net;AccountName=<YOUR-STORAGE-ACCOUNT-NAME>;AccountKey=<YOUR-STORAGE-ACCOUNT-KEY==>"
+  ```
+   
+
+3. **Add a new Azure site to the data_docs_sites section of your great_expectations.yml.**
+  
+    ```yaml
+  data_docs_sites:
+    local_site:
```
Comment (alexsherstinsky) [inline docs/guides/setup/configuring-data-docs/how-to-host-and-share-data-docs-on-azure-blob-storage.md]: The indentation did not render...

**Review by alexsherstinsky — CHANGES_REQUESTED** [review]
Indentation issues.

File: docs/guides/setup/configuring-data-docs/how-to-host-and-share-data-docs-on-azure-blob-storage.md (hunk 1 above)
Comment (cdkini) [reply docs/guides/setup/configuring-data-docs/how-to-host-and-share-data-docs-on-azure-blob-storage.md]: Oh great catch, thank you! Should be updated now

**Review by alexsherstinsky — APPROVED** [review]
LGTM

**Pull request merged**